redirects.yaml

```diff
@@ -683,3 +683,5 @@ usn/?: "https://usn.ubuntu.com/"
 (?P<page>.*)\.html: "/{page}"
 telco/?: "/telecommunications"
 telcos/?: "/telecommunications"
+kubernetes/docs/?: "http://blog.juju.solutions/kubernetes-docs/"
```

Comment:
Shouldn't we link to https?

Reply:
good point, I think that would be preferable

Reply:
Oh I didn't realise that works

Reply:
Fixed